<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>250</width>
                <height>137</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>GUI ScrcPy - Toolkit</string>
        </property>
        <property name="windowIcon">
            <iconset>
                <normaloff>rsrc/android_circle.png</normaloff>rsrc/android_circle.png
            </iconset>
        </property>
        <widget class="QPushButton" name="fullscreenUI">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>40</y>
                    <width>31</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="toolTip">
                <string>Fullscreen</string>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="icon">
                <iconset>
                    <normaloff>rsrc/monitor.png</normaloff>rsrc/monitor.png
                </iconset>
            </property>
        </widget>
        <widget class="QPushButton" name="clipD2PC">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>40</y>
                    <width>31</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="toolTip">
                <string>Copy device clipbioard to PC</string>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="icon">
                <iconset>
                    <normaloff>rsrc/clip.png</normaloff>rsrc/clip.png
                </iconset>
            </property>
        </widget>
        <widget class="QPushButton" name="powerUI">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>70</y>
                    <width>231</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="toolTip">
                <string>Power on/off</string>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="icon">
                <iconset>
                    <normaloff>rsrc/power.png</normaloff>rsrc/power.png
                </iconset>
            </property>
        </widget>
        <widget class="QPushButton" name="appswi">
            <property name="geometry">
                <rect>
                    <x>170</x>
                    <y>40</y>
                    <width>31</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="toolTip">
                <string>press the APP_SWITCH button</string>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="icon">
                <iconset>
                    <normaloff>rsrc/layout.png</normaloff>rsrc/layout.png
                </iconset>
            </property>
        </widget>
        <widget class="QPushButton" name="menuUI">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>100</y>
                    <width>71</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="toolTip">
                <string>Menu key</string>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="icon">
                <iconset>
                    <normaloff>rsrc/menu.png</normaloff>rsrc/menu.png
                </iconset>
            </property>
        </widget>
        <widget class="QPushButton" name="vup">
            <property name="enabled">
                <bool>false</bool>
            </property>
            <property name="geometry">
                <rect>
                    <x>210</x>
                    <y>10</y>
                    <width>31</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="toolTip">
                <string>Volume Up</string>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="icon">
                <iconset>
                    <normaloff>rsrc/volume.png</normaloff>rsrc/volume.png
                </iconset>
            </property>
        </widget>
        <widget class="QPushButton" name="back">
            <property name="geometry">
                <rect>
                    <x>170</x>
                    <y>100</y>
                    <width>71</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="toolTip">
                <string>Back key</string>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="icon">
                <iconset>
                    <normaloff>rsrc/left-arrow-1.png</normaloff>rsrc/left-arrow-1.png
                </iconset>
            </property>
        </widget>
        <widget class="QPushButton" name="home">
            <property name="geometry">
                <rect>
                    <x>90</x>
                    <y>100</y>
                    <width>71</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="toolTip">
                <string>Home key</string>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="icon">
                <iconset>
                    <normaloff>rsrc/home.png</normaloff>rsrc/home.png
                </iconset>
            </property>
        </widget>
        <widget class="QPushButton" name="vdown">
            <property name="enabled">
                <bool>false</bool>
            </property>
            <property name="geometry">
                <rect>
                    <x>210</x>
                    <y>40</y>
                    <width>31</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="icon">
                <iconset>
                    <normaloff>rsrc/volume.png</normaloff>rsrc/volume.png
                </iconset>
            </property>
        </widget>
        <widget class="QPushButton" name="screenfreeze">
            <property name="enabled">
                <bool>false</bool>
            </property>
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>191</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="text">
                <string>Screen Freeze</string>
            </property>
        </widget>
        <widget class="QPushButton" name="clipPC2D">
            <property name="geometry">
                <rect>
                    <x>90</x>
                    <y>40</y>
                    <width>31</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="toolTip">
                <string>Copy PC clipboard to Device</string>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="icon">
                <iconset>
                    <normaloff>rsrc/credit-card.png</normaloff>rsrc/credit-card.png
                </iconset>
            </property>
        </widget>
        <widget class="QPushButton" name="notif_pull">
            <property name="geometry">
                <rect>
                    <x>130</x>
                    <y>40</y>
                    <width>31</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="toolTip">
                <string>Expand notification panel</string>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="icon">
                <iconset>
                    <normaloff>rsrc/bell.png</normaloff>rsrc/bell.png
                </iconset>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
